cmd/setup-gh.go

```diff
@@ -81,9 +73,31 @@ application and service principle, and will configure that application to trust
 	return cmd
 }
 
-func fillSetUpConfig(sc *providers.SetUpCmd) error {
+func fillSetUpConfig(sc *providers.SetUpCmd, gh providers.GhClient) error {
+	if sc.TenantId == "" {
+		tenandId, err := providers.PromptTenantId(sc.AzClient, context.Background())
+		if err != nil {
+			return fmt.Errorf("prompting tenant ID: %w", err)
+		}
+		sc.TenantId = tenandId
+	}
+
 	if sc.AppName == "" {
-		sc.AppName = getAppName()
+		// make the default app name the current directory name plus "workflow" and a string
+
+		// get the current directory name
+		dir, err := os.Getwd()
+		if err != nil {
+			return fmt.Errorf("getting current directory: %w", err)
+		}
+		dirBase := filepath.Base(dir)
+		defaultAppName := dirBase + "-workflow"
```

Comment:
We also need to have the default app name adhere to the valid dns standards, all lowercase, no `_`, etc